tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +37,17 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert len(items) == 0
```

Comment:
In tests, I like the asserts to be as specific as we can make them. We are told this should return an empty list in this situation, therefore, the test should literally check for an empty list. This ensures the type is correct (a list) and that it has no contents.

Checking that the length of something is 0 misses checking for a list. An empty dictionary `items = {}` would also have a length of 0, but we know from the README that we expect a list.

```suggestion
    assert len(items) == []
```

Reply:
Updated the test to check for an empty list explicitly, ensuring the correct type and no contents.